helmfile.d/15-ory.yaml

```diff
@@ -0,0 +1,85 @@
+bases:
```

Comment:
Since Ory is needed for all installations let's add it to `10-base.yaml` instead of having its own separate file

Reply:
Can't the same be said about Management Portal? It is in its own file. As an alternative, we can rename 10-managementportal.yaml to 10-authentication.yaml or 10-identity-management.yaml and add both MP and Ory-Kratos. What do you think?

Reply:
In general I'd like to reduce number of helmfiles and merge them whenever it makes sense. I'm fine with changing the names, maybe we can merge both MP and Ory into 10-base.yaml.
If you won't want to put them into 10-base.yaml, since MP and appconfig aren't only about authentication or identity management I think we should have a more descriptive name.

Reply:
IMO, put it in `10-base.yaml`.

app-config is the most optional but you can still disable it. I don't think there is a big benefit to it being in a separate helmfile

Reply:
Merged into `10-base.yaml`!

Reply:
Because postgres is difined in the 10-managementporta.yaml file, I moved kratos to this helmfile.